src/supermarket/docs/CONFIGURING.md

```diff
@@ -1,60 +1,36 @@
 # Configuring Supermarket
 
-These instructions show how to configure Supermarket in development mode. If
-running in a production-like environment, you'll be using the Omnibus
-package install, where these settings are driven through
-/etc/supermarket/supermarket.rb.
-
-Supermarket uses [dotenv](https://github.com/bkeepers/dotenv) to handle setting
-environment variables. These are the different configurations needed to get
-Supermarket running. The default parameters provided in `.env` should be
-sufficient to run the test suite.
+These instructions show how to configure Supermarket in development mode. If running in a production-like environment, you'll be using the Omnibus package install, where these settings are driven through /etc/supermarket/supermarket.rb.
 
+Supermarket uses [dotenv](https://github.com/bkeepers/dotenv) to handle setting environment variables. These are the different configurations needed to get Supermarket running. The default parameters provided in `.env` should be sufficient to run the test suite.
 
 ## Recommended System Specs
 
 If you are running Supermarket in a production environment, these are the minimum, basic specifications we recommend for your Supermarket server:
-* At least 1 CPU
-* At least 1 GB Memory
-* At least 5 GB free space in /var
+
+- At least 1 CPU
```

Comment:
I know it's not new content, but this paragraph is weird. It's labelled "Recommended System Specs" and then "minimum, basic specification". If it's "Minimum system specs" then you could drop "At least" from each of these bullet points.